spec/fixtures/output/k8s/pod_yml_from_defaults.yml

```diff
@@ -0,0 +1,88 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  namespace: testuser
+  name: rspec-test-123
+  labels:
+    job: rspec-test-123
+    app.kubernetes.io/name: rspec-test
+    app.kubernetes.io/managed-by: open-ondemand
+    account: test
```

Comment:
I think with #224 there will be empty `annotations:` so tests may fail without that extra metadata key being here.

Reply:
Thanks! Yea I needed a rebase because merge conflict too, I think because that PR.

Anyhow - yea, I rebased and had to merge that.